apps/src/StudioApp.js

```diff
@@ -1142,6 +1143,27 @@ StudioApp.prototype.displayMissingBlockHints = function (blocks) {
 
 StudioApp.prototype.onReportComplete = function (response) {
   this.authoredHintsController_.finishHints(response);
+
+  if (!response) {
+    return;
+  }
+
+  // Track GA events
+  if (response.new_level_completed) {
+    trackEvent('Puzzle', 'Completed', response.level_path, response.level_attempts);
+  }
+
+  if (response.share_failure) {
+    trackEvent('Share', 'Failure', response.share_failure.type);
+  }
```

Comment:
Oh, I did not get the impression that these were useless, just that they were old. @ashercodeorg ?